test/example.test.js

```diff
@@ -1,55 +1,56 @@
 'use strict'
 
-const t = require('tap')
-const test = t.test
+const { test } = require('node:test')
 const { rimrafSync } = require('rimraf')
-const build = require('./example')
+const build = require('../examples/example')
```

Comment:
Well, we're still importing from those examples that's why

I don't know why they are mixed like this but if they are tests please leave them all in the tests folder, or completely separate tests and examples so we don't import them from the tests folder